core/src/main/resources/hudson/model/UpdateCenter/body.jelly

```diff
@@ -49,18 +49,15 @@ THE SOFTWARE.
             <l:icon class="icon-red icon-md"/>
             <l:icon class="icon-nobuilt icon-md"/>
           </div>
-          <p class="info jenkins-checkbox" style="font-weight: normal">
+          <p class="info">
             <j:if test="${app.lifecycle.canRestart()}">
               <j:if test="${app.isQuietingDown() or it.isRestartScheduled()}">
-                <input id="scheduleRestartCheckbox" type="checkbox" checked="checked" />
+                <f:checkbox checked="true" id="scheduleRestartCheckbox" name="scheduleRestart" title="${%warning}" />
               </j:if>
               <j:if test="${!app.isQuietingDown() and !it.isRestartScheduled()}">
-                <input id="scheduleRestartCheckbox" type="checkbox" />
+                <f:checkbox readonly="true" id="scheduleRestartCheckbox" name="scheduleRestart" title="${%warning}" />
```

Comment:
@NotMyFault Why did you (attempt to) make the checkbox here `readonly`? It doesn't work because https://github.com/jenkinsci/jenkins/commit/4cac031d81b4385410d1081548221ffefd1007c9#diff-6fbee721e02514a54ede8e6e5ed37d3af538239842117a33b7ee905b0a518c81R6 is wrong. Asking because I want to fix my bug and want to ensure I'm not introducing a different one.

Reply:
Good question, actually. I _think_ the attribute is obsolete and can be removed, given the if-condition represents the default state and the checkbox should work here.